### ISSUE TITLE
Cannot install  repository using yum with Chef client - version 12.14.60

### ISSUE THREAD

**Issue by AntoineBro** [DataDog/chef-datadog#360]
The latest release of the chef client include a  yum_repository provider ( see https://github.com/chef/chef/blob/master/lib/chef/provider/yum_repository.rb )
 
That causes DataDog install to fail :

```
  [2016-09-15T17:37:45+00:00] WARN: Chef::Provider::YumRepository already exists!  Cannot create deprecation class for LWRP provider yum_repository from cookbook yum
   [2016-09-15T17:37:45+00:00] WARN: YumRepository already exists!  Deprecation class overwrites Custom resource yum_repository from cookbook yum
...
         
         
         NoMethodError
         -------------
         undefined method `url' for Chef::Resource::YumRepository

         Cookbook Trace:
         ---------------
           /tmp/kitchen/cache/cookbooks/datadog/recipes/repository.rb:43:in `block in from_file'
           /tmp/kitchen/cache/cookbooks/datadog/recipes/repository.rb:40:in `from_file'
           /tmp/kitchen/cache/cookbooks/datadog/recipes/_install-linux.rb:21:in `from_file'
           /tmp/kitchen/cache/cookbooks/datadog/recipes/dd-agent.rb:26:in `from_file'
           /tmp/kitchen/cache/cookbooks/cs-chef-datadog-wrapper/recipes/other-platforms.rb:1:in `from_file'
           /tmp/kitchen/cache/cookbooks/cs-chef-datadog-wrapper/recipes/default.rb:18:in `from_file'
           /tmp/kitchen/cache/cookbooks/cs-chef-base-wrapper/recipes/default.rb:20:in `from_file'

         Relevant File Content:
         ----------------------
         /tmp/kitchen/cache/cookbooks/datadog/recipes/repository.rb:

          36:
          37:  when 'rhel', 'fedora'
          38:    include_recipe 'yum'
          39:
          40:    yum_repository 'datadog' do
          41:      name 'datadog'
          42:      description 'datadog'
          43>>     url node['datadog']['yumrepo']
          44:      proxy node['datadog']['yumrepo_proxy']
          45:      proxy_username node['datadog']['yumrepo_proxy_username']
          46:      proxy_password node['datadog']['yumrepo_proxy_password']
          47:      gpgkey node['datadog']['yumrepo_gpgkey']
          48:      action :add
          49:    end
          50:  end
          51:

         Platform:
         ---------
         x86_64-linux
```

**Comment by bai** [comment]
It should be `baseurl` instead of `url`.

**Comment by historus** [comment]
I already opened a PR https://github.com/DataDog/chef-datadog/pull/361

**Comment by olivielpeau** [comment]
Closed via #361
